scripts/build/build.mk

```diff
@@ -51,7 +51,7 @@ ifeq (legacy,$(findstring legacy,$(COSMOS_BUILD_OPTIONS)))
 endif
 
 ifeq (v2,$(findstring v2,$(COSMOS_BUILD_OPTIONS)))
-  SIMAPP = simapp/v2
+  SIMAPP = simapp
```

Comment:
hum, is this really correct? I do not think so, the dir of simappv2 is `simapp/v2`.

Reply:
oh. this is targetting 0.52. `COSMOS_BUILD_OPTIONS=v2 make install` should basically echo that simapp/v2 has been removed from release/v0.52.x. Otherwise it may be confusing that it just builds the normal simapp

cc @auricom